File: test-unittest/src/main/java/com/tistory/ykyh/test_unittest/MainActivity.java
package com.tistory.ykyh.test_unittest;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private EditText etHeight;
    private EditText etWeight;
    private Button btCalc;
    private TextView tvResult;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


    }
}

File: kotlinstudy/src/main/kotlin/com/ykyahwa/kotlinstudy/MainActivity.kt
package com.ykyahwa.kotlinstudy

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.LinearLayoutManager
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        bt_submit.setOnClickListener {
            tv_message.text = "Hello, " + et_name.text.toString()
        }

        rv_list.adapter = CityAdapter()
        rv_list.layoutManager = LinearLayoutManager(this)
        rv_list.adapter.notifyDataSetChanged()
    }

    override fun onStart() {
        super.onStart()
    }

    override fun onResume() {
        super.onResume()
    }

    override fun onPause() {
        super.onPause()
    }

    override fun onStop() {
        super.onStop()
    }

    override fun onDestroy() {
        super.onDestroy()
    }
}
